api/controllers/csv.js

```diff
@@ -26,9 +67,7 @@ exports.getAllHoursPastWeeksCSV = function (req, res, next) {
             $gte: weeksSinceEpoch() - req.params.num
         }
     }, function (err, docs) {
```

Comment:
Is this change part of the new Amida style enforcement, or just a personal thing? I don't particularly care, just curious if there is a reason for it.